components/src/core/AppBar/AppBar.tsx

```diff
@@ -1,50 +1,53 @@
 import React, { useCallback, useEffect, useState } from 'react';
-import { Theme, useTheme } from '@mui/material/styles';
-import createStyles from '@mui/styles/createStyles';
-import makeStyles from '@mui/styles/makeStyles';
+import { styled, useTheme } from '@mui/material/styles';
 import useMediaQuery from '@mui/material/useMediaQuery';
 import MuiAppBar, { AppBarProps as MuiAppBarProps } from '@mui/material/AppBar';
 // import { useScrollPosition } from '@n8tb1t/use-scroll-position';
 import clsx from 'clsx';
```

Comment:
@daileytj I see you've removed the clsx dependency from the package.json, do we still have access to it through a transitive dependency? 

Considering that it builds in CI, I think this should be fine.